lib/target/users/user.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Target.Users.User do
+  @moduledoc "User Module"
+
+  use Ecto.Schema
+  use Pow.Ecto.Schema
+
+  schema "users" do
+    pow_user_fields()
+    field :gender, :string, default: "other"
+
+    timestamps()
+  end
+
+  def changeset(user_or_changeset, attrs) do
+    user_or_changeset
+    |> pow_changeset(attrs)
+    |> changeset_gender(attrs)
+  end
+
+  def changeset_gender(user_or_changeset, attrs) do
```

Comment:
Why this is a different changeset? 🤔

Reply:
Well, took the idea from [here](https://hexdocs.pm/pow/user_roles.html#content) 
I guess that `changeset` is like the official one where you keep _validation titles_, and then you keep each validation modularized on its own changeset. Thoughts on this? @brunvez @mcousillas6

Reply:
Yes, you can have more than one changeset, for example: separate edit/create changesets to limit which fields can be updated.

Changesets are just an abstraction to group field updates for a record, so there is no limit on how many you can or should have

Reply:
Cool 👍

Reply:
The idea of many changesets is better if you use them in different places for different validations. For example, an `admin_changeset` that validates the role being equal to "admin". Since this is only called from within the other changeset I'd say it doesn't really fulfill that purpose. That being said, I don't mind leaving it if the function is private